cloudbuild.yaml

```diff
@@ -118,7 +100,7 @@ steps:
   # Run clang-tidy after the examples finish running, since they share the same `output_base`.
   - name: 'gcr.io/oak-ci/oak:latest'
     id: run_clang_tidy
-    waitFor: ['run_example_hello_world_nodejs']
+    waitFor: ['run_examples_cpp']
```

Comment:
It may be called `run_examples_with_cpp_app`, since it may be confusing what `cpp` stands for.